### ISSUE TITLE
The SED is  no longer in the child table of a Parent, the Parent still respond Data Request(802.15.4) with ACK(802.15.4). Is this behavior as expected?

### ISSUE THREAD

**Issue by jinpeng1989** [openthread/openthread#8406]

**Comment by jwhui** [comment]
Yes, this is expected. IEEE 802.15.4 ACKs are generated at the MAC layer without any input from higher layers.

If a child wants to validate that it is still in its parent's child table, it needs to have a protocol exchange at a higher layer - for example, something that involves exchanging IPv6 datagrams.

**Comment by abtink** [comment]
As Jonathan said, yes, expected  behavior. 

You can check  "[Child Supervision](https://openthread.io/guides/build/features/child-supervision)" feature in OT as a mechansim related to this.
